flask_dance/consumer/oauth1.py

```diff
@@ -197,7 +197,15 @@ def authorized(self):
             next_url = url_for(self.redirect_to)
         else:
             next_url = "/"
-        self.session.parse_authorization_response(request.url)
+            
+        try:
+            self.session.parse_authorization_response(request.url)
+        except Exception as err:
```

Comment:
It seems like this should raise a `TokenMissing` exception: https://github.com/requests/requests-oauthlib/blob/master/requests_oauthlib/oauth1_session.py#L352

Can you change this generic `Exception` type to that more specific `TokenMissing` exception?